src/EnergyPlus/DataSizing.cc

```diff
@@ -621,14 +622,16 @@ Real64 ZoneAirDistributionData::calculateEz(EnergyPlusData &state, int const Zon
     return zoneEz;
 }
 
-Real64 calcDesignSpecificationOutdoorAir(EnergyPlusData &state,
-                                         int const DSOAPtr,           // Pointer to DesignSpecification:OutdoorAir object
-                                         int const ActualZoneNum,     // Zone index
-                                         bool const UseOccSchFlag,    // Zone occupancy schedule will be used instead of using total zone occupancy
-                                         bool const UseMinOASchFlag,  // Use min OA schedule in DesignSpecification:OutdoorAir object
-                                         bool const PerPersonNotSet,  // when calculation should not include occupants (e.g., dual duct)
-                                         bool const MaxOAVolFlowFlag, // TRUE when calculation uses occupancy schedule  (e.g., dual duct)
-                                         int const spaceNum)
+Real64
+calcDesignSpecificationOutdoorAir(EnergyPlusData &state,
+                                  int const DSOAPtr,           // Pointer to DesignSpecification:OutdoorAir object
+                                  int const ActualZoneNum,     // Zone index
+                                  bool const UseOccSchFlag,    // Zone occupancy schedule will be used instead of using total zone occupancy
+                                  bool const UseMinOASchFlag,  // Use min OA schedule in DesignSpecification:OutdoorAir object
+                                  bool const PerPersonNotSet,  // when calculation should not include occupants (e.g., dual duct)
+                                  bool const MaxOAVolFlowFlag, // TRUE when calculation uses occupancy schedule  (e.g., dual duct)
+                                  int const spaceNum,          // Space index (if applicable)
+                                  bool const calcIAQMethods)   // For IAQProcedure, PCOccSch, and PCDesOcc, calculate if true, return zero if false
```

Comment:
New argument for `calcDesignSpecificationOutdoorAir` to return zero for certain methods.